static/js/TopicPage.jsx

```diff
@@ -633,7 +608,7 @@ TopicLink.propTypes = {
 };
 
 
-const TopicSideColumn = ({ slug, links, clearAndSetTopic, parashaData, tref, setNavTopic, timePeriod, properties, topicTitle, multiPanel }) => {
+const TopicSideColumn = ({ slug, links, clearAndSetTopic, parashaData, tref, setNavTopic, timePeriod, properties, topicTitle, multiPanel, topicImage }) => {
```

Comment:
Same as above, though that pattern seems to predate this PR...

Reply:
Following up here - as per our discussion, we are considering leaving this as a tech-debt chore.